rich/text.py

```diff
@@ -171,6 +171,20 @@ def __eq__(self, other: object) -> bool:
             return NotImplemented
         return self.plain == other.plain and self._spans == other._spans
 
+    def __lt__(self, other: object) -> bool:
+        if isinstance(other, str):
+            return self.plain < other
+        elif isinstance(other, Text):
+            return self.plain < other.plain
+        return False
```

Comment:
This should return `NotImplemeted` rather than False.

Reply:
Done.  I originally did this as a shortcut for comparing with None, but that is bad form.  I also removed the test cases accordingly.